client/src/App/index.js

```diff
@@ -1,23 +1,24 @@
-import React from 'react';
-import logo from '../logo.svg';
+import React, { useState, useMemo } from 'react';
 import './App.css';
+import { Switch, Route } from 'react-router-dom';
+import LoginPage from '../pages/logIn';
+import IsAuthContext from '../components/context/isAuthContext';
 
 function App() {
+  const [isAuth, setIsAuth] = useState(false);
+
+  const value = useMemo(() => ({ isAuth, setIsAuth }), [isAuth, setIsAuth]);
+
   return (
     <div className="App">
       <header className="App-header">
```

Comment:
Don't forget to pull the new version of the main and remove these two elements
```suggestion
```

Reply:
done